Dockerfile

```
@@ -34,20 +38,12 @@
       libxml2-dev \
       swig && \
-    sudo apt-get autoremove -y && \
-    sudo apt-get autoclean && \
-    sudo apt-get clean && \
-    sudo rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+    sudo apt-get autoclean
 
-WORKDIR /app
-COPY gems.rb gemfiles capybara-screenshot-diff.gemspec /app/
-COPY lib/capybara/screenshot/diff/version.rb /app/lib/capybara/screenshot/diff/
+RUN sudo sed -i 's/true/false/g' /etc/fonts/conf.d/10-antialias.conf
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)